phpstan.neon

```diff
@@ -11,6 +11,7 @@ parameters:
 		- docs
 		- lang
 		- wordpress (?)
+		- includes/forms/WC_Order.php
```

Comment:
There are a couple of phpstubs that need to be updated before we can include this file.
https://github.com/stellarwp/php-stubs-woocommerce-subscriptions/issues/1